### ISSUE TITLE
PNG support for urls in CSS modules

### ISSUE THREAD

**Issue by chadwilcomb** [HenrikJoreteg/hjs-webpack#28]
First off, I am loving this app setup from the human javascript videos. Great job on those (and this), @HenrikJoreteg 

In my project, I needed to add the following loader in order to get my app to compile with some PNGs that were referenced in some CSS files `{ test: /\.png/, loader: 'url?limit=100000&mimetype=image/png' }` 

This seems like a common use case, should the PNG loader be added by default? Or should we have the option to add additional loaders in the webpack.config options?

**Comment by lukekarrys** [comment]
I think it would be a good idea to add a default loader for `png` (and other) images.

@chadwilcomb Did you find it necessary to have `mimetype=image/png` in there? Looking at the code for [`url-loader`](https://github.com/webpack/url-loader/blob/master/index.js#L14) I think it will lookup the mimetype automatically, which would allow us to specify just one loader for all images:

```js
{
  test: /\.(jpe?g|png|gif)/,
  loader: 'url-loader?limit=10000'
}
```

**Comment by lukekarrys** [comment]
I opened #30 to discuss the code further. :smile:

**Comment by chadwilcomb** [comment]
Yeah, that works for me. I had just found that png loader example somewhere online and used it (I am very new to webpack).

**Comment by lukekarrys** [comment]
Hey @chadwilcomb you should be able to find this functionality in the newly published `v2.4.0`. Thanks again for the issue!

**Comment by chadwilcomb** [comment]
:+1: thanks!
